subgraphs/exchange/package.json

```diff
@@ -14,7 +14,7 @@
         "deploy-local": "graph deploy --node http://localhost:8020/ --ipfs http://localhost:5001 /sushiswap/sushiswap",
 
         "deploy:mainnet": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ sushiswap/exchange exchange.yaml",
-        "deploy:fantom": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ sushiswap/fantom-exchange exchange.yaml",
+        "deploy:fantom": "graph deploy --access-token 478f4210cc65456da8e7b4f39248c900 --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ sushiswap/fantom-exchange exchange.yaml",
```

Comment:
Probably want to double check if it's still valid ~

Reply:
Appreciate the heads up. Yeah I messed this up, but caught it right away and regenerated a new key.